src/commands/init/generateTemplate/index.js

```diff
@@ -73,7 +73,7 @@ function filterFiles(filters) {
 function render(template, data, callback) {
     let rendered;
     try {
-        rendered = Handlebars.compile(template)(data);
+        rendered = Handlebars.compile(template, data)(data);
```

Comment:
What do you think about adding `{ noEscape: true }` directly here, it is something we actually always want to use and means that we do not need to "pollute" overload `data`.